5.5/test/run

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+#
+# Test the MySQL image.
+#
+# IMAGE_NAME specifies a name of the candidate image used for testing.
+# The image has to be available before this script is executed.
+#
+
+set nounset
+
+IMAGE_NAME=${IMAGE_NAME-openshift/mysql-55-centos7-candidate}
+
+get_container_ip() {
+	docker inspect --format='{{.NetworkSettings.IPAddress}}' $CONTAINER_ID
+}
+
+mysql_cmd() {
+	mysql --host $(get_container_ip) -uuser -ppass "$@" db
+}
+
+cleanup() {
+	CONTAINER=$1 ; shift
+	echo "Cleaning up..."
+	docker stop $CONTAINER >/dev/null
+	docker rm $CONTAINER >/dev/null
+	echo "Done."
+}
+
+test_connection() {
+	echo "  Testing MySQL connection to $(get_container_ip)..."
+	local max_attempts=10
+	local sleep_time=1
+	local attempt=1
+	while [ $attempt -le $max_attempts ]; do
+		echo "    Trying to connect..."
+		set +e
+		mysql_cmd <<< "SELECT 1;" >/dev/null 2>&1
+		status=$?
+		set -e
+		if [ $status -eq 0 ]; then
+			echo "  Success!"
+			return 0
+		fi
+		attempt=$(( $attempt + 1 ))
+		sleep $sleep_time
+	done
+	echo "  Giving up: Failed to connect"
+	return 1
+}
+
+test_mysql() {
+	echo "  Testing table creation"
```

Comment:
You're actually doing create + insert + select + drop, so maybe some other comment than table creation?